### PR TITLE
Frogger-game

### PR DESCRIPTION
**Pull request by ASTkach**
### Frogger-game

[Demo](https://astkach.github.io/frontend-nanodegree-arcade-game/) |
[Code base](https://github.com/ASTkach/frontend-nanodegree-arcade-game/tree/frogger-game)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by ASTkach** [comment]
Hope I did a good self-check.

File: submissions/ASTkach/frogger-game/app.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+Enemy.prototype.position();
+
+Enemy.prototype.collision = function () {
+    if (
+        !(
+            player.x > this.x + CELL_WIDTH - 10 ||
+            player.x + CELL_WIDTH - 10 < this.x ||
+            player.y > this.y + CELL_HEIGHT - 10 ||
+            player.y + CELL_HEIGHT - 10 < this.y
```
Comment (OleksiyRudenko) [inline submissions/ASTkach/frogger-game/app.js:108]: Bot's requirements re OOP addresses exactly this situation.
Please double check and fix as appropriate.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@ASTkach very well done!
However one important fix is yet required.

**Comment by OleksiyRudenko** [comment]
Up

**Pull request closed without merging**